internal/compiler/fileloader.go

```diff
@@ -266,7 +280,10 @@ func (p *fileLoader) addAutomaticTypeDirectiveTasks() {
 		containingDirectory = p.opts.Host.GetCurrentDirectory()
 	}
 	containingFileName := tspath.CombinePaths(containingDirectory, module.InferredTypesContainingFile)
-	p.rootTasks = append(p.rootTasks, &parseTask{normalizedFilePath: containingFileName, isForAutomaticTypeDirective: true})
+	p.rootTasks = append(p.rootTasks, &parseTask{
+		normalizedFilePath:          containingFileName,
+		isForAutomaticTypeDirective: true,
+	})
```

Comment:
The include reason is missing here, so probably this would crash in the editor?

Reply:
no its not needed because its not used . let me fix this